### DISCUSSION TITLE
v2.8.6

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4535] [分享 / Share]
✨新增
- `清爽首页` / `极简首页` 现在可以正确重定向 "悬浮视频". (#4404)
- `网址参数清理` 增加 `spmid` 参数. (#4512)
- `快捷键扩展` 默认不再启用前进/后退的快捷键 (默认使用 b 站的前进/后退). (#4501)
- `展开动态内容` 现在不会展开专栏类型的动态, 并保留原来的 "全文" 按钮. (#4475)

🐛修复
- 修复了首页的正式版安装链接 CDN 不正确. (#4460)
- 修复 `极简首页` 在特定情况下无法触发加载的问题. (#4302, PR #4430 by [Tinhone](https://github.com/Tinhone))
- 调整了 `简化评论区` 和 `删除广告` 对评论区附近的活动横幅和小喇叭横幅的说明, 关于这两种元素的具体区别以及分配至的组件, 可以参考 #4444 中的讨论.
- 修复 `直播信息扩充` 中的标题未处理转义的 HTML 特殊字符.
- 修复 `BV 号转换` 复制的链接即使没有参数也会带上末尾的 `?`. (#4515)
- 修复番剧片单页面出现报错. (#1999)
- 修复 `自定义顶栏` 历史面板无法正确跳转课程. (#4484)
- 修复 `自定义顶栏` 在历史记录页面产生的布局错位. (#4459)
- 修复 `网址参数清理` 处理相对路径 URL 时产生了错误的结果. (#4471)
- 修复 `自定义顶栏` 在专栏页面挡住了作者信息顶栏.

☕开发者相关
- 锁定了 PR Check 中的 pnpm 版本, 避免 pnpm 更新后出现不兼容 lockfile 导致 PR Check 失败.
- `夜间模式` 增加了一篇详细的[开发文档](./registry/lib/components/style/dark-mode/README.md).
- Microsoft Edge 不再作为首要兼容的浏览器. (和 Chrome 相同内核, 理论上兼容性不会有太大差别)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.8.6'>v2.8.6</a>.</em>

**Comment by Geeyun-JY3** [comment]
`快捷键扩展`为啥后退也默认不启用？

**Reply by the1812** [reply]
这是成对的操作, 只禁用一个不是很奇怪吗 (变成后退可全页面触发, 前进又必须聚焦播放器)

**Reply by Geeyun-JY3** [reply]
只禁用一个又确实是有点怪，行吧（

**Reply by lvxuanwu** [reply]
从前天开始我这里出现番剧无法观看的情况，其他视频正常，今天更新2.8.7依然无法观看
![QQ截图20231123125541](https://github.com/the1812/Bilibili-Evolved/assets/73596222/1e89d2fb-43a5-42a9-8500-c2ad5b3cd293)

**Reply by Geeyun-JY3** [reply]
@lvxuanwu 反馈 bug 建议开 issue 按照模板填提供更多信息，仅仅发在某个 release 对应的 discussion 下（且不应发在 discussion 的不相关讨论串下，如此处我在 #4535 发起的这个讨论串）会使相关反馈分散，并不利于开发者排查解决问题。

**Comment by Geeyun-JY3** [comment]
#4302 是没被自动关掉🤔？

**Reply by the1812** [reply]
我手动关了, 不知道 PR 里那个 fix 为啥没生效
